service.go

```diff
@@ -168,8 +168,17 @@ type (
 
 	// MaintenanceWindow during which maintenance operations should take place
 	MaintenanceWindow struct {
-		DayOfWeek string `json:"dow"`
-		TimeOfDay string `json:"time"`
+		DayOfWeek string               `json:"dow"`
+		TimeOfDay string               `json:"time"`
+		Updates   []*MaintenanceUpdate `json:"updates,omitempty"`
+	}
+
+	// MaintenanceUpdate represents a maintenance needing to be applied on the service.
+	MaintenanceUpdate struct {
+		Deadline    *string `json:"deadline"`
```

Comment:
please mark all the properties of this struct as `omitempty`

Reply:
it's done :)